### PR TITLE
Repo restructure

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: cardano-node/app/wallet-client.hs [hunk 1]
```diff
@@ -12,10 +12,10 @@ import           Cardano.Node.Configuration.Types (CardanoConfiguration (..),
                                                    CardanoEnvironment (..))
 import           Cardano.Node.Features.Logging (LoggingCLIArguments (..),
                                                 LoggingLayer (..),
-                                                createLoggingFeature,
-                                                loggingParser)
-import           Cardano.Prelude hiding (option)
+                                                createLoggingFeature
+                                                )
```
Comment (intricate) [inline cardano-node/app/wallet-client.hs]: Doesn't matter, really, but perhaps this ")" should be on the same line as `createLoggingFeature`.

File: cardano-node/app/chairman.hs [hunk 2]
```diff
@@ -21,8 +20,42 @@ import           Ouroboros.Consensus.Node.ProtocolInfo.Abstract (NumCoreNodes (.
...
+                          caMaxBlockNo
+                          stdoutTracer
+
+    case caTimeout of
+      Nothing      -> run
+      Just timeout ->
+        run
+        `race_`
+        do
+          threadDelay (timeout * 1_000_000)
```
Comment (intricate) [inline cardano-node/app/chairman.hs]: The `1_000_000` confused me initially but apparently this is something introduced by the `NumericUnderscores` extension.

File: cardano-cli/Main.hs [hunk 3]
```diff
@@ -1 +0,0 @@
-import Cardano.CLI.Run (main)
```
[the remark this answers is not in the document]
Comment (deepfire) [reply cardano-cli/Main.hs]: @intricate, just to clarify -- this was directly inspired by https://github.com/input-output-hk/cardano-node/issues/84#issue-474068961

File: cardano-node/README.md [hunk 4]
```diff
@@ -1,3 +1,80 @@
+
+# Shelley Testnet
+
+The `./scripts/shelley-testnet.sh` script starts up three nodes that are
```
[the remark this answers is not in the document]
Comment (Jimbo4350) [reply cardano-node/README.md]: Will focus on documentation in a later PR

File: cardano-node/app/wallet-client.hs (hunk 1 above)
Comment (Jimbo4350) [reply cardano-node/app/wallet-client.hs]: I actually need to figure out some formatting conventions :thinking: but I will also follow up on this in a later PR.

**Comment by mrBliss** [comment]
Nice!

File: cardano-node/app/wallet-client.hs (hunk 1 above)
Comment (mrBliss) [reply cardano-node/app/wallet-client.hs]: I suggest using `stylish-haskell`, that already takes care of the imports.

**Pull request merged**